### ISSUE TITLE
chainging the format of existing partition requires the Format value to be set

### ISSUE THREAD

**Issue by dbungert** [canonical/ubuntu-desktop-installer#1817]
Suppose an existing partition of filesystem type A, and we use manual partitioning to change that partition to filesystem type B.  In UI this is possible (unrelated - this shows some of the filesystem options we didn't want to offer) - but it doesn't enforce the format checkbox.

Attempting to proceed in this state will yield a "changing partition format requires a wipe value" traceback.  Please enforce the checkbox being set for that case.

**Comment by jpnurmi** [comment]
Changing in relation to the original config? If I go and change it back to A, should I be allowed to leave it unformatted?

**Comment by dbungert** [comment]
Yes, if the filesystem type is set back to the original value, A in this case, the format checkbox being checked is no longer required.

**Comment by jpnurmi** [comment]
Hmm, and presumably this only applies to preserved partitions? What about newly created partitions, are there any restrictions whether they must be formatted or can they be left unformatted?

**Comment by dbungert** [comment]
Yes, this is about preexisting partitions that would start off in a `preserved: true` state.  Newly created partitions are either going to be left unformatted, or have an implied wipe due to getting the mkfs tool run on them.
